dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/servlet/ServletWrapper.java

```diff
@@ -151,7 +151,6 @@
     protected IPlatformHelper platformHelper;
```

Comment:
@jhanders34 , also now unused import here for `import com.ibm.wsspi.webcontainer.collaborator.IConnectionCollaborator;`

Reply:
Done